qa/rpc-tests/sc_invalidate.py

```diff
@@ -67,380 +68,153 @@ def join_network(self):
         time.sleep(2)
         self.is_network_split = False
 
-    def dump_ordered_tips(self, tip_list):
```

Comment:
You probably move disconnect_nodes function to test framework utils, as you did for dump_sc_info_record and dump_ordered_tips. In fact disconnect_nodes I wonder why it was not already there.
If you move disconnect_nodes to framework utils, please make sure to remove the redefinition for all the other tests.